src/jit/lir.h

```diff
@@ -262,6 +262,10 @@ class LIR final
         void InsertBefore(GenTree* insertionPoint, GenTree* node);
         void InsertAfter(GenTree* insertionPoint, GenTree* node);
 
+        void InsertBefore(GenTree* insertionPoint, GenTree* node1, GenTree* node2);
```

Comment:
That's an interesting idea. Not an area of C++11 that I'm terribly familiar with :)

What are the performance implications there?